_data/students.yml

```diff
@@ -101,14 +101,21 @@
   github: aryankedare
   image: aryankedare.jpg
   twitter: KedareAryan
+  
+- name: Abhinav Raj
+  country: India
+  github: abhinavraj0123456789
+  image: AbhinavRaj.jpg
+  twitter: hardwired_raj
 
 - name: Samyak Jain
   github: samyakbvs
   image: samyak.png
   twitter: Samyakbvs
+  facebook: Samyak Jain 
```

Comment:
@abhinavraj0123456789 Extra line added. Kindly remove it.

Reply:
@abhinavraj0123456789 Just remove this line and your PR is ready to merge

Reply:
@abhinavraj0123456789 This facebook link of Samyak Jain is added twice, kindly update the PR